lib/Controller/ImageController.php

```diff
@@ -377,14 +378,75 @@ private function getImageJPEG($blob, $mimetype): array
         } catch (\ImagickException $e) {
             throw Exceptions::Forbidden('Imagick failed to read image: '.$e->getMessage());
         }
-
         // Convert to JPEG
         try {
             $image->autoOrient();
-            $image->setImageFormat('jpeg');
-            $image->setImageCompressionQuality(95);
+            $format = $this->config->getSystemValueString('memories.highres_format', 'jpeg');
+            switch ($format) {
+              case 'jpeg':
+                $format = 'jpeg';
+                break;
+              case 'webp':
+                $format = 'webp';
+                break;
+              /*case 'avif': //CPU Benchmark
+                //$format = 'avif';
+                break*/
+              default:
+                $format = 'jpeg';
+            }
```

Comment:
Don't need this check. If the admin sets the value to something invalid, it's not our problem.